tests/warn/i17493.scala

```diff
@@ -0,0 +1,5 @@
+//> using options -explain
+ class A(val s: String) extends AnyVal {
+  // def f = eq("hello, world") // no warning for now because `eq` is inlined
+  def g = synchronized { println("hello, world") } // warn
+ }
```

Comment:
Indentation issue:
```suggestion
//> using options -explain
class A(val s: String) extends AnyVal {
  // def f = eq("hello, world") // no warning for now because `eq` is inlined
  def g = synchronized { println("hello, world") } // warn
}
```